src/Catty/ViewController/MediaLibrary/MediaLibraryViewController.swift

```diff
@@ -42,7 +42,7 @@ final class MediaLibraryViewController: UICollectionViewController {
     private var originalAudioSessionCategoryOptions: AVAudioSessionCategoryOptions?
 
     private var audioPlayer: AVAudioPlayer?
-    private var audioPlayerDelegate: AudioPlayerFinishPlayingCompletionCaller?
+    private weak var audioPlayerDelegate: AudioPlayerFinishPlayingCompletionCaller?
```

Comment:
Setting this var _weak_ will lead to a crash when pressing the play button for a sound in the Media Library since the reference to the delegate's instance will be lost once the following method returns:

`soundsLibraryCollectionViewDataSource(_ dataSource: SoundsLibraryCollectionViewDataSource, didPlaySoundOf item: MediaItem, completion: (() -> Void)?)`